src/components/info/location-info.vue

```diff
@@ -0,0 +1,279 @@
+    clickCoordinateLuref.value = infos.clickCoordinateLuref
+    formattedCoordinates.value = Object.fromEntries(
+      Object.entries(INFO_PROJECTIONS).map(([crs, label]) => [
+        label,
+        formatCoords(location, map.getView().getProjection(), crs),
+      ])
+    )
+    elevation.value =
+      infos.elevation === null ? 'N/A' : formatElevation(infos.elevation)
+    address.value = infos.address
+    isInBoxOfLidar.value = infos.isInBoxOfLidar
+  }
+}
+
+watch(currentUser, user => {
+  userRole.value = user?.role || 'anonymous'
+  userType.value = user?.role || 'base'
+})
+
+const isRapportForageVirtuelAvailable = computed(() => userRole.value === 'ACT')
+const isCyclomediaAvailable = computed(
+  () =>
+    userType.value === 'etat' ||
+    userType.value === 'commune' ||
+    userRole.value === 'MinTour'
+)
+const isImagesObliquesAvailable = computed(() => true)
+
+const lidarUrl = computed(() =>
+  clickCoordinateLuref.value
+    ? `${import.meta.env.VITE_LIDAR_URL}?COORD_X=${
+        clickCoordinateLuref.value[0]
+      }&COORD_Y=${clickCoordinateLuref.value[1]}&COORD_Z=${parseInt(
+        elevation.value || '0'
+      )}`
+    : ''
+)
+const forageUrl = computed(() =>
+  //  "http://172.20.0.3:3000/x/print_sync?pdf=A6")
+  clickCoordinateLuref.value
+    ? `${import.meta.env.VITE_FORAGE_URL}?x=${
+        clickCoordinateLuref.value[0]
+      }&y=${clickCoordinateLuref.value[1]}`
+    : ''
+)
+const cyclomediaUrl = computed(() =>
+  clickCoordinateLuref.value
+    ? `${import.meta.env.VITE_CYCLOMEDIA_URL}?q=${
+        clickCoordinateLuref.value[0]
+      };${clickCoordinateLuref.value[1]}`
+    : ''
+)
+const imagesObliquesUrl = computed(() =>
+  clickCoordinateLuref.value
+    ? `${import.meta.env.VITE_OBLIQUE_URL}?x=${
+        clickCoordinateLuref.value[0]
+      }&y=${clickCoordinateLuref.value[1]}&crs=2169`
+    : ''
+)
+
+function addRoutePoint() {
+  if (!locationInfo.value) {
+    return
+  }
+  const point = new Feature(new Point(locationInfo.value))
+  if (address.value && address.value.distance <= 100) {
+    point.set('label', formatAddress(address.value))
+  } else {
+    point.set('label', formattedCoordinates.value['Luref'])
+  }
+  routingFeatureTemp.value = point
+}
+
+function toggleStreetview() {
+  isStreetviewActive.value = !isStreetviewActive.value
+}
+
+async function downloadRapportForageVirtuel() {
+  downloadingRepport.value = true
+  map.getViewport().style.cursor = 'wait'
+  try {
+    const response = await fetch(forageUrl.value)
+    if (!response.ok) {
+      throw new Error()
+    }
+    const blob = await response.blob()
+    const downloadUrl = URL.createObjectURL(blob)
+    const anchor = document.createElement('a')
+    anchor.href = downloadUrl
+    anchor.download = ''
+    anchor.click()
+    URL.revokeObjectURL(downloadUrl)
+  } catch {
+    // TODO harmonize error
+    alert('Error downloading forage')
+  } finally {
+    map.getViewport().style.cursor = ''
```

Comment:
move this in a watcher (watch downloadingRepport state)

Reply:
ok